### PR TITLE
Issue631 clean up installation rst

### LINKED ISSUE #631: Clean up INSTALLATION.rst
Remove obsolete information on Python(x,y), EPD, etc. from `INSTALLATION.rst`.

See: 
- [x] https://github.com/usnistgov/fipy/pull/607#pullrequestreview-224039046
- [x] https://github.com/usnistgov/fipy/pull/607#pullrequestreview-223931364
- [x] https://github.com/usnistgov/fipy/pull/607#pullrequestreview-224050911

### PR DESCRIPTION
**Pull request by guyer**
Fixes #631

### CODE REVIEW

File: INSTALLATION.rst [hunk 1]
```diff
@@ -48,55 +48,66 @@ Recommended Method
 
 .. attention::
 
-   There are many ways (described further down) to obtain the software
+   There are many ways to obtain the software
    packages necessary to run :term:`FiPy`, but the most expedient way is
-   with the conda_ package manager.
+   with the conda_ package manager.  In addition to the scientific
```
Comment (tkphd) [inline INSTALLATION.rst]: As a proper noun, Conda ought to be capitalized except when referring to `conda` explicitly.

File: INSTALLATION.rst [hunk 2]
```diff
@@ -448,7 +328,7 @@ directories. The reasons for this include:
 To avoid tampering with the system Python_ installation, you can employ one
 of the utilities that manage packages and their dependencies independently
 of the system package manager and the system directories.  These utilities
-include conda_, Stow_, Virtualenv_ and zc.buildout_, amongst others.
+include conda_, Nix_, Stow_, Virtualenv_ and zc.buildout_, amongst others.
```
Comment (tkphd) [inline INSTALLATION.rst]: [Buildout](https://pypi.org/project/zc.buildout/) refers to itself as Buildout, not zc.buildout.

File: INSTALLATION.rst (hunk 1 above)
Comment (guyer) [reply INSTALLATION.rst]: I squander no neurons trying to figure out how to capitalize things when they can't figure it out themselves.

File: INSTALLATION.rst (hunk 2 above)
Comment (guyer) [reply INSTALLATION.rst]: Why?!?

File: INSTALLATION.rst (hunk 1 above)
Comment (guyer) [reply INSTALLATION.rst]: AKA, find me a case where they call themselves "Conda" (and not "CONDA" or "conda") that's not at the beginning of a sentence and I might begin to care

File: INSTALLATION.rst [hunk 3]
```diff
@@ -48,55 +48,66 @@ Recommended Method
...
-       This command creates a self-contained conda_ environment and then
-       downloads and populates the environment with the prerequisites for
-       :term:`FiPy` from the channels guyer_ and conda-forge_ at
-       https://anaconda.org.
+  .. note::
 
-    .. attention::
+     This command creates a self-contained conda_ environment and then
+     downloads and populates the environment with the prerequisites for
+     :term:`FiPy` from the channels guyer_ and conda-forge_ at
```
Comment (wd15) [inline INSTALLATION.rst]: Does it still use the guyer channel?

File: INSTALLATION.rst [hunk 4]
```diff
@@ -448,7 +328,7 @@ directories. The reasons for this include:
 To avoid tampering with the system Python_ installation, you can employ one
 of the utilities that manage packages and their dependencies independently
 of the system package manager and the system directories.  These utilities
-include conda_, Stow_, Virtualenv_ and zc.buildout_, amongst others.
+include conda_, Nix_, Stow_, Virtualenv_ and Buildout_, amongst others.
 Conda_ is the only one of these we have the resources to support.
```
Comment (wd15) [inline INSTALLATION.rst]: Currently, we support both Conda and Nix (see below) builds of FiPy.

**Comment by wd15** [comment]
This is so much better now.

File: INSTALLATION.rst (hunk 1 above)
Comment (tkphd) [reply INSTALLATION.rst]: OK -- FiPy style is to take the lead of the entity referred to by the proper noun, not to blindly capitalize proper nouns as most style guides (including The Carpentries) would have us do.

File: INSTALLATION.rst (hunk 3 above)
Comment (guyer) [reply INSTALLATION.rst]: Nope

File: INSTALLATION.rst (hunk 4 above)
Comment (guyer) [reply INSTALLATION.rst]: *I* don't support Nix. Do you  support Nix?

File: INSTALLATION.rst (hunk 1 above)
Comment (guyer) [reply INSTALLATION.rst]: FiPy style is to be respectful of what entities choose to call themselves unless they make it a PITA to figure out what that is

**Pull request merged**